src/Types.jl

```diff
@@ -1478,7 +1478,7 @@ end
 # Find package by UUID in the manifest file
 # TODO this needs a better name now?
 function manifest_info(env::EnvCache, uuid::UUID)::Union{PackageEntry,Nothing}
-    uuid in values(env.uuids) || find_registered!(env, [uuid])
+    any(uuids -> uuid in uuids, values(env.uuids)) || find_registered!(env, [uuid])
```

Comment:
This was never true before, since `eltype(values(env.uuids))` is `Vector{UUID}` which is never equal to a `UUID`.

Reply:
I think that `env.uuids` was previously a `Dict{String,UUID}` but it's not anymore so this is now never true.